### ISSUE TITLE
Different game-play options

### ISSUE THREAD

**Issue by jai-dewani** [utkarsh-raj/chopGame#1]
I can divide my fingers even when one of my hand is dead.

**Comment by utkarsh-raj** [comment]
As I have understood the rules, it seems to me that this is feature and not an issue. It would be helpful if any of us has some verification regarding the same.

**Comment by jai-dewani** [comment]
Yes, it varies from place to place, it will be better if we could introduce various modes of the game.
